src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MongoDbSessionHandler.php

```diff
@@ -69,7 +69,7 @@ class MongoDbSessionHandler implements \SessionHandlerInterface
      */
     public function __construct($mongo, array $options)
     {
-        if (!($mongo instanceof \MongoClient || $mongo instanceof \Mongo)) {
+        if (!($mongo instanceof \MongoClient || $mongo instanceof \Mongo || $mongo instanceof \MongoDB\Client)) {
             throw new \InvalidArgumentException('MongoClient or Mongo instance required');
```

Comment:
$mongo instanceof \MongoDB\Client
-> MongoDB is available

any alternatives ?